example/Makefile

```diff
@@ -1,21 +1,22 @@
 
-KUBERNETES_VERSION=1.21.1
-KUSTOMIZE_VERSION=3.8.9
-TOPOLVM_VERSION=0.8.3
+KUBERNETES_VERSION ?= 1.21.1
+KUSTOMIZE_VERSION := 3.8.9
+TOPOLVM_VERSION := 0.8.3
 
-KIND_CLUSTER_NAME=topolvm-example
+KIND_CLUSTER_NAME ?= topolvm-example
 
-SUDO=sudo
-BINDIR=$(shell pwd)/bin
-KUSTOMIZE=$(BINDIR)/kustomize
-KUBECTL=$(BINDIR)/kubectl
-KIND=../bin/kind
+SUDO := sudo
+BINDIR := $(shell pwd)/bin
+TMPDIR := /tmp/topolvm
```

Comment:
Do you need to use `?=`?

Reply:
I confirmed that no problem caused using `:=` instead of `?=` in this Makefile. I replaced all `?=` to `:=`.

Reply:
The behavior is described here:  https://www.gnu.org/software/make/manual/make.html#Overriding